packages/components/inputs/localized-rich-text-input/src/localized-rich-text-input.visualspec.js

```diff
@@ -40,6 +55,8 @@ describe('LocalizedRichTextInput', () => {
     const doc = await getDocument(page);
     let input = await queries.findByTestId(doc, 'rich-text-data-test-en');
 
+    await input.focus();
```

Comment:
There is a change in the default behavior of the slate editor. The way it works now is that editor without any prior focus assumes that point [0,0] is selected. We might toggle the marks but the results won't be visible until the editor gets the focus.
![slate-default-selection](https://user-images.githubusercontent.com/49066275/161492312-3df7b1c1-7ea2-430d-b84f-edf988dd12fe.gif)

After the changes the editor has to explicitly receive focus to apply the initial selection.